.github/workflows/containers.yml

```diff
@@ -0,0 +1,76 @@
+name: Upload Container image
+
+on:
+  schedule:
+  - cron: "0 */12 * * *"
+  push:
+    branches:
+     - master
+    tags:
+      - 'v*'
```

Comment:
Looks like this would get triggered too often, I'd say either 'master' or (better) the 'v' tag should be sufficient. Running this every 12 hours sounds 'no matter what' sounds a bit much...

Reply:
The schedule will only run against the master branch. Push against master is required to test the master branch prior to creating a release.